system/datatypes/mailinglist.php

```diff
@@ -442,6 +452,77 @@ private function unsubscribeAction(HyphaRequest $request) {
 		return ['redirect', isset($_GET['redirect']) ? $_GET['redirect'] : $this->path()];
 	}
 
+	/**
+	 * @param HyphaRequest $request
+	 * @return null|array
+	 */
+	private function remindAction(HyphaRequest $request) {
+		$code = isset($_GET['code']) ? $_GET['code'] : null;
```

Comment:
Should this authenticate using a code? Why not just check `isUser()`? AFAIU there is never a need for a non-logged-in user to send reminders, so why use a code here? That only leads to additional exposure of the code, which can actually allow (semi-malicious) logged in users to craft a confirmation url, bypassing the usual consent-mechanism for subscriptions.

Reply:
Reading again I see that the code is also used to identify the e-mail address to remind, but then I would rather pass the e-mailaddress to confirm.

Reply:
fixed